hosting/views.py

```diff
@@ -924,7 +927,19 @@ def get_success_url(self):
     def get(self, request, *args, **kwargs):
         vm = self.get_object()
         if vm is None:
-            return redirect(reverse('hosting:virtual_machines'))
+            if self.request.is_ajax():
+                storage = messages.get_messages(request)
+                for m in storage:
```

Comment:
Not sure why would we need to loop through messages.

Reply:
in the `get_object()` method for this view, we are adding an error message when a VM is not found. and we are confirming the termination of a VM when it is not found. So to clear the message, according to the docs, the messages instance has to be iterated.
https://docs.djangoproject.com/en/1.11/ref/contrib/messages/#expiration-of-messages